content/post/career_ladders.md

```diff
@@ -0,0 +1,97 @@
+
+## How to Create a Career Ladder
+I separate a good career ladder into two parts: guiding principles and specific competencies.
+
+### Guiding Principles
+Guiding principles are simple heuristics that capture how roles differ from each other. You don’t get promoted for just getting better at the same job, you get promoted for showing that you can do a different job.
+
+Some examples of guiding principles:
+- A general management heuristic we use at [Yerdle](https://www.yerdlerecommerce.com/about-us): a Manager executes a program with direction, while a Director can create a program from scratch with little oversight.
+- Software Engineering at [Microsoft](https://www.quora.com/What-are-the-differences-between-SDE-SDE-II-and-Senior-SDE-at-Microsoft): a Software Engineer creates complex solutions to simple problems, a Software Engineer II creates simple solutions to simple problems, a Senior Software Engineer creates simple solutions to complex problems, and a Principal Software Engineer makes complex problems disappear.
+- Design at [Lyst](https://medium.com/designing-lyst/career-progression-for-product-designers-user-researchers-at-lyst-49c0c6cb9003): Junior contributes to the work, Mid does the work, Senior leads the work, and Lead leads the team.
+- Data Science, according to [Jan Zawadzki](https://towardsdatascience.com/becoming-a-level-3-0-data-scientist-52641ff73cb3): Level 1 is statistics skills, Level 2 adds engineering skills, and Level 3 adds business skills.
+
+When I think about how analytics team members work at different levels and how to clearly articulate the difference between roles, I focus on two areas: scope of influence (how wide the impact of your work is), and what deliverables look like (the types of work you do, and what done looks like). Here is my matrix for those principles for individual contributors:
+{{< figure src="/img/annotations-google-sheet.jpg" title="Guiding principles for individual contributors">}}
+
+Guiding principles set the stage for the specific competencies that comprise the rest of the ladder. But importantly, they also enable you to create flexibility and variations of the ladder as needed. You can modify a career ladder for various flavors - for example, I’ve had members of my team who strongly wanted to follow a specialist path. I manage fairly small teams and am happy to accommodate different career paths; knowing the guiding principles allowed me to create a Product Analyst ladder and a Sales Operations Analyst ladder that spoke to their skills more specifically, but still aligned very clearly with our overall levelling strategy.
```

Comment:
> our overall levelling strategy.

I think you're referring to "the company's" overall levelling strategy, in which case you can clarify to

> the company's overall levelling strategy.